conftest.py

```diff
@@ -0,0 +1,37 @@
+from Matrix import Matrix
+import pytest
+
+@pytest.fixture
+def add_result():
+    f_m = Matrix([[0, 1],[0, 2]])
+    s_m = Matrix([[0, 1],[0, 2]])
```

Comment:
Do not use short-named variables like `f_m` or `a_b`, because there are not understandable by other than you developers